.vs/config/applicationhost.config

```diff
@@ -0,0 +1,1020 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Actually, VS makes several new files and changes existing ones, I think mainly because I install the `System.Reflection.Metadata` nuget package for completions. I'm not sure what our policy is with respect to files for VS though.